tests/integration/azure/test_access.py

```diff
@@ -94,6 +98,81 @@ def test_create_global_spn(skip_if_not_in_debug, env_or_skip, az_cli_ctx, make_c
     )
 
 
+@pytest.fixture
+def clean_warehouse_config(env_or_skip, az_cli_ctx) -> Generator[None, None, None]:
+    """Clean workspace warehouse configuration."""
+    env_or_skip("IDE_PROJECT_ROOTS")  # Only run from editor
```

Comment:
What do we think about only running this in the editor not in the CI?

- Pro: The failing `test_create_global_spn` went unnoticed because it is not ran in the CI. To be more precise, I cannot run that integration test as I miss permissions to change Azure resources. It also put me on the wrong track because I *assumed* it was working, letting me to think the issue solved by this PR was due to a context different then our testing environment. Running the integration test more often will make similar issues more clear.
- Con: we are changing a global config that will interfere with other tests when they run in parallel. Especially now the tests uses dummy resources thus blocking the warehouse to start for a moment as the config is invalid